extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -101,6 +101,13 @@ private function getUserById( $userId ) {
 			throw new \InvalidArgumentException( self::NO_USER_MATCH_ERROR );
 		}
 
+		$user->load();
+
+		// Anonymous users can't have contributions
+		if ($user->mName === false) {
```

Comment:
Styling convention including spacing in MW/php app follows the [MediaWiki guidelines](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP).